homeassistant/components/comelit/sensor.py

```diff
@@ -167,4 +167,4 @@ def native_value(self) -> StateType:
         if (status := self._zone_object.human_status) == AlarmZoneState.UNKNOWN:
             return None
 
-        return cast(StateType, status.value)
+        return cast(str, status.value)
```

Comment:
Do we need to cast it?

Reply:
Without

`homeassistant/components/comelit/sensor.py:170: error: Returning Any from function declared to return "str | int | float | None"  [no-any-return]`

Reply:
changing the return type to `str`:

`homeassistant/components/comelit/sensor.py:170: error: Returning Any from function declared to return "str"  [no-any-return]`

I see the issue in the `status.value` that is seen as `Any`

Reply:
I think ` AlarmZoneState` in the library should be made a `StrEnum`.

Reply:
Let's keep it like this for now.